academy/ibc/migration.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "IBC Migration"
+order: 
+description: 
+tag: deep-dive
+---
+
+# IBC Migration
+
+When working with IBC, it is important to be able to upgrade IBC chains and clients. IBC-connected chains must perform an IBC upgrade if their upgrade will break counterparty IBC clients. This is of importance for all developers working with Tendermint clients enabling upgrades.
+
+The **IBC module** serves several different user groups, such as:
```

Comment:
talk a bit about the versioning that ibc-go uses, this is relevant for understanding migrations & upgrades:

https://github.com/cosmos/ibc-go/blob/main/RELEASES.md